std_msgs/Float64 rho_1
std_msgs/Float64 rho_2
std_msgs/Float64 rho_3
